Felienne/hedy
# Fix: Set maximum amount of "try it" attempts when logged in

Provide a general summary of your changes in the Title of the PR

**Description**

Currently, users are able to try the example programs unlimited times. But, as pointed out by @MarleenGilsing, students no longer create their own programs. In this PR we implement a counter to restrict logged in users to only try out three different example codes per level, enforcing them to write their own code after looking at some examples.

**Fix for**

This PR fixed #1176.

**How to test**

Notice the different view on the code page when being logged in or not. When logged in a header should show with information on how many attempts of example code you have left. When reaching 0 the commands window on the left side will disappear. If, for some reason, the student still tries to copy example code a pop-up warning will appear. In the case of not being logged in the UI and functionality should be identical as before this PR. See screenshot below for clarification:

![Schermafbeelding 2021-11-05 om 18 39 14](https://user-images.githubusercontent.com/38155424/140554629-f410f02c-35bd-4e38-ad54-6136ef6af7ae.png)

![Schermafbeelding 2021-11-05 om 18 39 23](https://user-images.githubusercontent.com/38155424/140554640-1fa92f0b-d1df-4ac4-b99e-9d962d6007bb.png)


**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
